qutebrowser/config/configtypes.py

```diff
@@ -798,6 +799,18 @@ class File(BaseType):
 
     typestr = 'file'
 
+    def transform(self, value):
+        if not value:
+            return None
+        value = os.path.expanduser(value)
+        value = os.path.expandvars(value)
+        if not os.path.isabs(value):
+            if standarddir.config():
```

Comment:
Please explicitely check for `... is not None` here - in this case it wouldn't matter as it never should be anything else falsey (i.e. it'd never return an empty string), but it still makes it more clear this is expected to be a string or `None`.